src/parser/jobs/drk/modules/MultiHitSkills.js

```diff
@@ -0,0 +1,65 @@
+import ACTIONS from 'data/ACTIONS'
+import Module from 'parser/core/Module'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import {Plural, Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import React from 'react'
+
+const REQUIRED_MULTI_HIT_TARGETS = {
+	[ACTIONS.FLOOD_OF_SHADOW.id]: 2,
+	[ACTIONS.UNMEND.id]: 2,
+	[ACTIONS.STALWART_SOUL.id]: 2,
+	[ACTIONS.QUIETUS.id]: 3,
+}
+
+const SEVERITY_LOW_TARGET_ATTACKS = {
+	1: SEVERITY.MEDIUM,
```

Comment:
1 as `MEDIUM` seems a bit harsh, but maybe that's just me. Perhaps they hit an aoe but the add died before the damage snapshot.

Reply:
That makes sense to me.  1 MINOR, 2 MEDIUM, 4 MAJOR?  or maybe 3 medium/6 major?  I kind of feel like this module won't really cater much to our core users anyway -- if this is flagging on you, there's probably a lot going on with the log anyway.